generate_web_reports.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+#
+# Generate the web report source codes (*.html and *.js) for the web report
+
+set -e
+
+function show_help {
+    echo "Usage: ./generate_build_web_reports.sh [tag1] [tag2] [linux_repo_root_url]"
+    echo "Example: ./generate_build_web_reports.sh tag1 tag2 linux_url"
+    echo
+    echo "Options:"
+    echo "  tag1    Required: old_linux_version [e.g. v6.8]"
+    echo "  tag2    Required: new_linux_version [e.g. v6.9]"
+    echo "  -h      Display this help message"
+    exit 0
+}
+
+if [[ "$1" == "-h" ]]; then
+    show_help
+fi
+
+# Ensure required arguments (tag1 and tag2) are provided
+if [ -z "$1" ] || [ -z "$2" ] || [ -z "$3" ]; then
+  echo "Usage: $0 <tag1> <tag2> <linux_repo_root_url>"
+  exit 1
+fi
+
+TAG1="$1"
+TAG2="$2"
+REPO_URL="$3"
+
+./web_scripts/generate_data.sh "$TAG1" "$TAG2" "$REPO_URL"
```

Comment:
Will this break if someone re-arranges your project repository or changes the `web_scripts` directory name?

Reply:
- repo_url is a user-defined repository link for cloning the Linux source code directory.
- Changing the web_scripts directory name will break the functionality. However, this is not considered an issue since the user who changes the name (part of the delta-kernel source code) should also modify the path name accordingly.